src/functions.sh

```diff
@@ -77,22 +133,22 @@ has_shebang () {
   local file="$1"
 
   # shell shebangs detection
-  if head -n1 "${file}" | grep -E '^\s*((\#|\!)|(\#\s*\!)|(\!\s*\#))\s*(\/usr(\/local)?)?\/bin\/(env\s+)?(sh|ash|bash|dash|ksh|bats)\b'; then
+  if head -n1 "${file}" | grep -E '^\s*((\#|\!)|(\#\s*\!)|(\!\s*\#))\s*(\/usr(\/local)?)?\/bin\/(env\s+)?(sh|ash|bash|dash|ksh|bats)\b' > /dev/null; then
     return 0
   fi
 
   # ShellCheck shell directive detection
-  if grep -E '^\s*\#\s*shellcheck\s+shell=(sh|ash|bash|dash|ksh|bats)\s*' "${file}"; then
+  if grep -E '^\s*\#\s*shellcheck\s+shell=(sh|ash|bash|dash|ksh|bats)\s*' "${file}" > /dev/null; then
```

Comment:
`grep -q` should quiet the standard output.

Reply:
Thanks, that is a better solution.